src/hb-ot-color-cbdt-table.hh

```diff
@@ -506,9 +506,7 @@ struct CBDT
 
     inline hb_blob_t* reference_blob_for_glyph (hb_codepoint_t  glyph_id,
 						unsigned int    requested_x_ppem,
-						unsigned int    requested_y_ppem,
-						unsigned int   *strike_x_ppem,
```

Comment:
I think we still want both x/y so we choose best.  Or at least, we need the max of two.  Don't call it "requested_y_ppem".

Reply:
cbdt logic is what was before me on get_extents.

Reply:
> cbdt logic table matching is what was before me on get_extents.

I know.  We should improve it.